ext/src/ruby_api/execution_error.rs

```diff
@@ -45,17 +74,27 @@ impl ExecutionError {
         let ruby_h = rhash_with_capacity(2);
         ruby_h.aset("path", self.path())?;
         ruby_h.aset("message", self.message())?;
+        if let Some(_i) = &self.locations {
+            let location_hashes: Vec<RHash> = self
+                .locations
+                .unwrap()
```

Comment:
The `unwrap` can be avoided like this:
```suggestion
        if let Some(locations) = &self.locations {
            let location_hashes: Vec<RHash> = locations
```

Reply:
:+1:, thanks, done!